components/WarningNote/WarningNoteRecap/WarningNoteRecap.tsx

```diff
@@ -32,29 +66,41 @@ const WarningNoteRecap = ({
     warningNote.warningNoteReviews.length > 0 &&
     formName != 'Warning Note Created';
 
+  const reverseDisplayOrderOfReviews = warningNote.warningNoteReviews
+    .slice(0)
```

Comment:
What's this `slice` doing?

Reply:
See conversation about `splice()` [in this PR](https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/391#discussion_r643814814)
N.B: This was an attempt to pull it out into a more readable format.